app/shared/productTypes.tsx

```diff
@@ -94,16 +103,35 @@ export interface HolidayStopFlowProperties {
   };
 }
 
+export interface DeliveryProblemType {
+  label: string;
+  messageIsMandatory: boolean;
+}
+
+export const commonDeliveryProblemTypes: DeliveryProblemType[] = [
+  { label: "Damaged Paper", messageIsMandatory: true },
+  { label: "Delivered Despite Holiday", messageIsMandatory: false },
```

Comment:
it doesn't look like this option allows the user to tick holiday stop delivery records which would make sense
speaking to @JessL123 we should just drop this option for the time being and she's raising a ticket to reinstate it once we've figured out how it should work (inc. w.r.t. crediting etc.)